cmake/FindXp.cmake

```diff
@@ -0,0 +1,17 @@
+find_package(Xext)
+find_package(Xau)
+
+find_package(PkgConfig)
+pkg_check_modules(PC QUIET xp)
```

Comment:
I believe use of the Xp library was deprecated in #30 so FindXp/linking to libXp shouldn't be needed... I could be proven wrong though

Reply:
This depends on the OS.  In Debian and Ubuntu, libXp is not needed, but it is in Red Hat based systems (openmotif) depends on it.  I am not sure about Gentoo.

Reply:
libXp has been deprecated on Gentoo.